src/main/scala/xiangshan/frontend/IFU.scala

```diff
@@ -657,48 +654,50 @@ class NewIFU(implicit p: Parameters) extends XSModule
     }
 
     is(m_sendTLB){
-      when( io.iTLBInter.req.valid && !io.iTLBInter.resp.bits.miss ){
-        mmio_state := m_tlbResp
-      }
+      mmio_state := Mux(io.iTLBInter.req.fire, m_tlbResp, m_sendTLB)
     }
 
     is(m_tlbResp){
-      val tlbExept = io.iTLBInter.resp.bits.excp(0).pf.instr ||
-                     io.iTLBInter.resp.bits.excp(0).af.instr ||
-                     io.iTLBInter.resp.bits.excp(0).gpf.instr
-      mmio_state         := Mux(tlbExept, m_waitCommit, m_sendPMP)
-      mmio_resend_addr   := io.iTLBInter.resp.bits.paddr(0)
-      mmio_resend_af     := mmio_resend_af || io.iTLBInter.resp.bits.excp(0).af.instr
-      mmio_resend_pf     := mmio_resend_pf || io.iTLBInter.resp.bits.excp(0).pf.instr
-      mmio_resend_gpf    := mmio_resend_gpf || io.iTLBInter.resp.bits.excp(0).gpf.instr
-      mmio_resend_gpaddr := io.iTLBInter.resp.bits.gpaddr(0)
+      when(io.iTLBInter.resp.fire) {
+        // we are using a blocked tlb, so resp.fire must have !resp.bits.miss
+        assert(!io.iTLBInter.resp.bits.miss, "blocked mode iTLB miss when resp.fire")
```

Comment:
Grammar mistake with regard to "miss".

Reply:
Should this be fixed? Or we can Resolve this conversation.